crux-mir/install.project

```diff
@@ -0,0 +1,8 @@
+packages:
+  ../crucible/
+  ../crux/
+  ./
+
+optional-packages:
+  ../dependencies/aig/
+  ../dependencies/what4/what4/
```

Comment:
Are these the only packages needed even to run the test suite?

Reply:
Running the test suite with this is probably not actually necessary, given what it's used for, so never mind.

Reply:
Right, though it turns out the test suite does actually work with `cabal v2-test --project-file=install.project`